modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -401,7 +403,10 @@ void RenderableOrbitalKepler::initializeGL() {
         _trailProgram->uniformLocation("modelViewTransform");
     _uniformTrailCache.projection =
         _trailProgram->uniformLocation("projectionTransform");
-    _uniformTrailCache.trailFade = _trailProgram->uniformLocation("trailFade");
+    _uniformTrailCache.colorFadeCutoffPoint =
+        _trailProgram->uniformLocation("colorFadeCutoffPoint");
```

Comment:
Since this code deals with rendering trails and points, how about calling this `colorFadeCutoffLevel` or `colorFadeCutoffValue` instead?

Reply:
Changed to `colorFadeCutoffValue`